src/lib/ngsiNotify/Notifier.cpp

```diff
@@ -338,6 +337,7 @@ static std::vector<SenderThreadParams*>* buildSenderParamsCustom
     params->renderFormat     = renderFormatToString(renderFormat);
     params->fiwareCorrelator = fiwareCorrelator;
     params->extraHeaders     = headers;
+    params->registration     = false;
```

Comment:
Not that I particularly like the name I picked, but my intention was to be generic. Probably soon registrations will have the same functionality as well and a variable with a very specific name will then be worthless. A variable to just differentiate ngsi9 from ngsi10 seems more useful.

Let's sleep on it ... :-)